.github/workflows/publish.yaml

```diff
@@ -0,0 +1,56 @@
+name: Build & Publish
+on: [workflow_dispatch]
+
+jobs:
+  build:
+    name: release
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '16.20.1'
+    # Run install dependencies
+    - name: Install dependencies
+      run: npm install
+    # Run tests
+    - name: Build 
+      run: npm run compile
+    - name: Get current package version
+      id: package_version
+      uses: martinbeentjes/npm-get-version-action@v1.1.0
+    - name: Check version is mentioned in Changelog
+      id: changelog_reader
+      uses: mindsers/changelog-reader-action@v2
+      with:
+        validation_depth: 10
+        version: ${{ steps.package_version.outputs.current-version }}
+        path: 'CHANGELOG.md'
+    - name: Create a Release
+      id: create_release
+      uses: actions/create-release@v1
+      env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} 
+      with:
+        tag_name : ${{ steps.package_version.outputs.current-version}}
+        release_name: ${{ steps.package_version.outputs.current-version}}
+        body: Publish ${{ steps.changelog_reader.outputs.changes }}
+    - name: Create vsix and publish to marketplace
+      id: create_vsix
+      uses: HaaLeo/publish-vscode-extension@v0
+      with:
+        pat: ${{ secrets.VS_MARKETPLACE_TOKEN }}
+        registryUrl: https://marketplace.visualstudio.com
+    - name: Attach vsix to release
```

Comment:
💡Just for archival reasons: I presume this was stock standard way this repo was getting released previously? Thanks  heaps,

Reply:
Yes, I am not sure how this was released previously but it was under different org in marketplace. Maybe it was released via internal pipelines.

Reply:
💡☕️ thanks!! cool, in that case it might be beneficial to definitely figure out the way it was released, and how artefacts were used. Just to be safe about it. Could be we can reach out to @danv-msft  for kind help and if he remember how this was released so we have process in place which is same as previously followed or we document what changed.

Reply:
If we are going to properly publish this - do you think we should change the publisher from `mindaro-dev`?

Reply:
@daniv-msft Can you help us if you still remember what was the release process for this tool ? Also if you can check if this PR is not missing any required steps please ?

@ross-p-smith I don't think we need to change the publisher, changing them would be losing all comments, stars and downloads until now from marketplace, we went through similar process for another tool, and we decided not to do it and it was not an easy process. Hope this make sense. Thank you.

Reply:
Sorry for the slow reply. We released this extension manually, i.e. using "vsce publish" as described here: https://code.visualstudio.com/api/working-with-extensions/publishing-extension
Now if you're adding the publish workflow, I don't know if it becomes immediately usable to actually release that change once merged? That would be a good first try.

Reply:
@daniv-msft Yes it will be immediately usable to publish this extension.